paasta_tools/kubernetes_tools.py

```diff
@@ -2229,7 +2229,14 @@ def get_all_namespaces(kube_client: KubeClient) -> List[str]:
 
 def ensure_namespace(kube_client: KubeClient, namespace: str) -> None:
     paasta_namespace = V1Namespace(
-        metadata=V1ObjectMeta(name=namespace, labels={"name": namespace})
+        metadata=V1ObjectMeta(
+            name=namespace,
+            labels={
+                "name": namespace,
+                "yelp.com/owner": "compute_infra_platform_experience",
+                "paasta.yelp.com/managed": "true",
```

Comment:
might also be worth using `paasta_prefixed` here to generate the label name (and/or maybe extract it to a constant since we're referring to it in this file and in paasta_secrets_sync)